functions.php

```diff
@@ -36,4 +36,85 @@ function displayDisease(array $data): string {
         $htmlToOutput = $htmlToOutput . $htmlForRow;
     }
     return $htmlToOutput;
-}
\ No newline at end of file
+}
+
+/**
+ * Securely add user input to database
+ */
+function addNewDiseaseToDB()
+{
+    $db = new PDO('mysql:host=db;dbname=Disease_db', 'root', 'password');
+
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+    $query = $db->prepare('INSERT INTO `disease_table` (`Organism`, `Incubation_usual`, `Incubation_range`, `Symptoms`, `Severity`, `Avg_annual_incidence`) 
+VALUES (:organism, :incubation_usual, :incubation_range, :symptoms, :severity, :avg_annual_incidence)');
+
+    $query->execute(['organism' => $_POST['organism'],
+        'incubation_usual' => $_POST['incubation_usual'],
+        'incubation_range' => $_POST['incubation_range'],
+        'symptoms' => $_POST['symptoms'],
+        'severity' => $_POST['severity'],
+        'avg_annual_incidence' => $_POST['avg_annual_incidence']]);
+}
+
+
+/**
+ * Grab all organism key-value pairs from a multidimensional array
```

Comment:
Maybe say why you're doing this (i.e. creating an array of organism names from which you'll be able to select in order to delete organisms from the db?)